sxhkd-mode.el

```diff
@@ -44,6 +44,9 @@ Can be t, nil or ask"
                 (const :tag "Ask" ask))
   :group 'sxhkd-mode)
 
+(defvar sxhkd-mode-indentation-length 2
+  "Indentation length")
```

Comment:
This one should be defcustom:
```
(defcustom sxhkd-mode-indentation-length 2
  "Indentation length."
  :type 'integer
  :group 'sxhkd-mode)
```

Reply:
Thanks for the review, it is fixed in last commit.